x-pack/dockerlogbeat/handlers.go

```diff
@@ -26,6 +28,26 @@ type StopLoggingRequest struct {
 	File string
 }
 
+// CapabilitiesResponse represents the response to a capabilities request
+type CapabilitiesResponse struct {
```

Comment:
I don't see this type being used outside this package. Let's keep it private to this package.